src/app/services/graph-client/msal-service.ts

```diff
@@ -18,13 +18,26 @@ export async function logIn(): Promise<any> {
       const authResponse = await msalApplication.acquireTokenSilent(loginRequest);
       return authResponse;
     } catch (error) {
-      return null;
+      if (requiresInteraction(error)) {
+        return acquireTokenWIthInteraction(loginRequest);
+       } else {
+         return null;
```

Comment:
If an error happens we won't be able to track it down because we are returning null, instead, throw the error for now. We'll need to think about handling errors gracefully.